app/src/main/java/de/tum/in/tumcampusapp/utils/DateTimeUtils.kt

```diff
@@ -13,13 +13,13 @@ object DateTimeUtils {
     /**
      * Format an upcoming string nicely by being more precise as time comes closer
      */
-    fun getFutureTime(time: DateTime, context: Context): String {
+    fun printFutureTime(time: DateTime, context: Context): String {
```

Comment:
I’m not sure that renaming it from `getFutureTime()` to `printFutureTime()` is good. It’s **not** printing anything and still does not tell me what the function does. Maybe it shoud be something like `getRelativeFutureTime()`? Also, please provide example results in the documentation.

Reply:
@thellmund I think `get*()` is really misleading, since those names are usually used for members or member-like functions and that doesn't really apply for this usage.

I agree, `print*()` isn't good either, that was just what JodaTime used in their DateTimeFormatter, so I initially stuck to that naming...
Now it's `formatFutureTime()`, which hopefully better describes what it does